demisto_sdk/commands/postman_codegen/postman_codegen.py

```diff
@@ -447,3 +450,23 @@ def duplicate_requests_check(commands_names_dict: dict) -> None:
     assert len(duplicates_list) == 0, f'There are requests with non-unique names: {duplicates_list}.\n' \
                                       f'You should give a unique name to each request.\n' \
                                       f'Names are case-insensitive and whitespaces are ignored.'
+
+
+def find_shared_args_path(flattened_json: dict) -> dict:
+    names_dict: dict = defaultdict(list)
+    shared_args_path: dict = defaultdict(int)
+    for key in flattened_json:
+        path_split = key.split('.')
+        arg_name = path_split[-1].lower()
+
+        # finds the maximum shared path with all other arguments in the same list
+        for other_arg in names_dict[arg_name]:
+            other_path_split = other_arg.split('.')
+            for max_same_path, (other_path, arg_path) in enumerate(zip(other_path_split[::-1], path_split[::-1])):
```

Comment:
This function is looking for a minimum split point from 0 until there's a difference in the arguments' path even if the max_same_path for this group was already changed from 0 by an earlier check. You should first get the existing max_same_path value and just check if there's a need for a longer path.